systemtest/src/test/java/io/strimzi/systemtest/log/LogSettingST.java

```diff
@@ -412,6 +421,37 @@ void testBridgeLogSetting(ExtensionContext extensionContext) {
         checkContainersHaveProcessOneAsTini(bridgeName);
     }
 
+    @ParallelTest
+    // This test might be flaky, as it gets real logs from CruiseControl pod
+    void testCruiseControlLogChange() {
+        final String debugText = " DEBUG ";
+        String cruiseControlPodName = PodUtils.getPodNameByPrefix(LOG_SETTING_CLUSTER_NAME + "-" + Constants.CRUISE_CONTROL_CONTAINER_NAME);
+        LOGGER.info("Check that default/actual root logging level is info");
+        String containerLogLevel = cmdKubeClient().execInPod(cruiseControlPodName, "grep", "-i", "rootlogger.level",
+                Constants.CRUISE_CONTROL_LOG_FILE_PATH).out().trim().split("=")[1];
+        assertThat(containerLogLevel.toUpperCase(Locale.ENGLISH), is(not(debugText.strip())));
+
+        LOGGER.info("Check logs in CruiseControl - make sure no DEBUG is found there.");
+        String logOut = StUtils.getLogFromPodByTime(cruiseControlPodName, Constants.CRUISE_CONTROL_CONTAINER_NAME, "20s");
+        assertThat(logOut.toUpperCase(Locale.ENGLISH), not(containsString(debugText)));
+
+        InlineLogging logging = new InlineLogging();
+        logging.setLoggers(Collections.singletonMap("rootLogger.level", debugText.strip()));
+        KafkaResource.replaceKafkaResource(LOG_SETTING_CLUSTER_NAME, kafka -> kafka.getSpec().getCruiseControl().setLogging(logging));
+
+        LOGGER.info("Wait for change of root logger in {}.", cruiseControlPodName);
+        TestUtils.waitFor("Waiting for log to be changed", Duration.ofSeconds(5).toMillis(), CO_OPERATION_TIMEOUT_MEDIUM, () -> {
```

Comment:
Maybe you can use some constant?

Reply:
`Duration.ofSeconds(5).toMillis()` but the name would be inappropriate.. I searched for them, but I did not like any :-)

Reply:
so create new one?

Reply:
done.